src/routers/datatypeRouters/edges/genes_genes.ts

```diff
@@ -112,22 +116,15 @@ async function findGenesGenes (input: paramsFormatType): Promise<any[]> {
   `
 
   const query = `
-    LET geneNodes = (
-      FOR record IN ${genesSchema.db_collection_name as string}
-      FILTER ${geneFilters}
-      RETURN record._id
-    )
-
-    FOR record IN ${genesGenesSchema.db_collection_name as string}
-    FILTER (record._from IN geneNodes OR record._to IN geneNodes) ${filters} ${arrayFilters}
-    SORT record._key
-    LIMIT ${input.page as number * limit}, ${limit}
-    RETURN MERGE({
-      'gene 1': ${verbose ? `(${sourceVerboseQuery})` : 'record._from'},
-      'gene 2': ${verbose ? `(${targetVerboseQuery})` : 'record._to'}},
-      (record.source == 'CoXPresdb' ? {${getDBReturnStatements(CoXPresdbSchema)}} : {${getDBReturnStatements(genesGenesSchema)}}))
-  `
-
+      FOR record IN ${genesGenesSchema.db_collection_name as string}
+      FILTER (record._from IN ['${geneIDs.join('\', \'')}'] OR record._to IN ['${geneIDs.join('\', \'')}']) ${filters} ${arrayFilters}
+      SORT record._key
+      LIMIT ${Number(input.page) * limit}, ${limit}
+      RETURN MERGE({
+        'gene 1': ${verbose ? `(${sourceVerboseQuery})` : 'record._from'},
+        'gene 2': ${verbose ? `(${targetVerboseQuery})` : 'record._to'}},
+        (record.source == 'CoXPresdb' ? {${getDBReturnStatements(CoXPresdbSchema)}} : {${getDBReturnStatements(genesGenesSchema)}}))
+    `.toString()
```

Comment:
we don't need this `toString()`, did the new eslint complain about it?

Reply:
That is so strange. It must be the editor that added it somehow. I will change it back.